k-distribution/tests/regression-new/checks/checkCircularList.k.out

```diff
@@ -1,2 +1,2 @@
 [Error] Compiler: Had 1 parsing errors.
-[Error] Compiler: Illegal circular relation: Exp < ExpList < Exp
+[Error] Compiler: Illegal circular relation: ExpList < Exp < ExpList
```

Comment:
The output is unstable because it's based on the iteration order of `relations.entrySet()`, but I don't see a clean way to address this.

Reply:
We can revisit this in the future if it becomes a worse problem; this code is pretty stable and so long as we aren't doing further big changes here we should be fine!

Reply:
If we care more that there's a circularity, and less what the circularity is, we could `sed` out the example given in the error message.